src/Cassandra/Tasks/TaskHelper.cs

```diff
@@ -195,6 +195,18 @@ private static Exception PreserveStackTrace(Exception ex)
         /// <summary>
         /// Smart ContinueWith
         /// </summary>
+        public static Task Continue<TIn>(this Task<TIn> task, Action<Task<TIn>> next)
```

Comment:
This method is not necessary, it is used by `ChangeKeyspaceAsync()` but it can be safely removed as `task.Continue(rs => Keyspace = keyspace);` returns a `Task<string>` which is still a `Task`.